extensions/sql-database-projects/src/common/utils.ts

```diff
@@ -45,3 +48,29 @@ export function trimChars(input: string, chars: string): string {
 
 	return output;
 }
+
+/**
+ * get quoted path to be used in any commandline argument
+ * @param filePath
+ */
+export function getSafePath(filePath: string): string {
+	return (os.platform() === 'win32') ?
+		getSafeWindowsPath(filePath) :
+		getSafeNonWindowsPath(filePath);
+}
+
+/**
+ * ensure that path with spaces are handles correctly
+ */
+export function getSafeWindowsPath(filePath: string): string {
+	filePath = filePath.replace('\\', '\\\\').replace('"', '');
```

Comment:
IIRC, javascript's string.replace will only replace the first occurrence. you have to use Regular Expression.